frontend/src/scenes/funnels/useFunnelTooltip.tsx

```diff
@@ -116,7 +116,8 @@ export function useFunnelTooltip(showPersonsModal: boolean): React.RefObject<HTM
         }
         const tooltipRect = tooltipEl.getBoundingClientRect()
         if (tooltipOrigin) {
-            ReactDOM.render(
+            // TODO: Shouldn't we unmount at some point??
+            createRoot(tooltipEl).render(
```

Comment:
This legacy architecture of the insight tooltip is horrible, and React notices… any time the tooltip rendered, this shows up in the console:
`Warning: You are calling ReactDOMClient.createRoot() on a container that has already been passed to createRoot() before.`
In the world map it's easy to rack up a thousand of these warnings, so that's probably the last thing we should solve before shipping this upgrade

Reply:
Would something like this work? https://github.com/PostHog/posthog/pull/17731/commits/69ed960277cf149cdc09c793a88f536b3892d583

Tooltips seem to render fine and no console errors 🎆 
![Oct-31-2023 18-51-57](https://github.com/PostHog/posthog/assets/6685876/6db04cde-c548-46f7-acec-bba23a767d5a)